database/src/main/runs/future/write_measurement.sql

```diff
@@ -13,13 +13,13 @@
  * limitations under the License.
  */
 
+
 CREATE OR REPLACE FUNCTION runs.write_measurement(
-    IN  i_segmentation          HSTORE,
-    IN  i_id_measurement        UUID,
+    IN  i_segmentation          JSONB,
     IN  i_id_checkpoint         UUID,
```

Comment:
hmm is `i_id_measurement` not needed here anymore?

Reply:
oh okay I think this function is far from being finished

Reply:
Thanks I missed this one being unfinished 😊

Reply:
Ah, it's in the future dir.